src/network/networkd-wiphy.c

```diff
@@ -271,3 +322,46 @@ int manager_udev_process_wiphy(Manager *m, sd_device *device) {
 
         return 0;
 }
+
+int manager_udev_process_rfkill(Manager *m, sd_device *device) {
+        _cleanup_free_ char *parent_path = NULL, *parent_name = NULL;
+        sd_device_action_t action;
+        const char *s;
+        Wiphy *w;
+        int r;
+
+        assert(m);
+        assert(device);
+
+        r = sd_device_get_action(device, &action);
+        if (r < 0)
+                return log_device_warning_errno(device, r, "Failed to get udev action, ignoring: %m");
+
+        r = sd_device_get_syspath(device, &s);
+        if (r < 0)
+                return log_device_warning_errno(device, r, "Failed to get syspath, ignoring: %m");
+
+        /* Do not use sd_device_get_parent() here, as this is a 'remove' uevent. */
```

Comment:
It might be any event no? Shouldn't this say, "as this might be a 'remove' uevent" ?